core/modules/views_ui/wizards/views_ui_base_views_wizard.php

```diff
@@ -687,9 +687,9 @@ protected function default_display_filters_user($form, $form_state) {
 
     if (!empty($form_state['values']['show']['type']) && $form_state['values']['show']['type'] != 'all') {
       $bundle_key = $this->entity_info['bundle keys']['bundle'];
```

Comment:
I think we just need to be checking `entity keys` here instead of `bundle keys`, and then we'll get the right key value from `taxonomy_entity_info()`.

Reply:
Thanks @quicksketch! Great improvement.